gotocompany/optimus/core/v1beta1/replay.proto

```diff
@@ -30,6 +30,7 @@ service ReplayService {
       body: "*"
     };
   }
```

Comment:
@deryrahman even though the request schema is exactly the same, I believe you still need to create a new message which match the rpc name: `ReplayDryRunRequest`

Reply:
As we want to make the request same with actual replay create, I think we don't need to make a new message. Any reason why we need to create new message?